packages/styles/README.md

```diff
@@ -0,0 +1,130 @@
+# @synergy-design-system/styles
+
+This package provides easy to use standalone helper classes based on the [Synergy Design Tokens](https://synergy-design-system.github.io/?path=/docs/packages-tokens--docs).
+
+---
+
+## 1. Installation
+
+Please make sure to install the package as a dependency:
+
+```bash
+npm install --save @synergy-design-system/styles
+
+# Install the required design tokens
+# (only needed if you do not install peerDeps automatically)
+npm install --save @synergy-design-system/tokens
+```
+
+---
+
+## 2. Loading the provided stylesheets
+
+### 2.1 Loading all styles (Recommended)
+
+To load all provided css classes at once, just include the default export into your application.
+It contains all styles found in the `src` directory.
+
+```html
+<!DOCTYPE html>
+  <head>
+    <link rel="stylesheet" href="/node_modules/@synergy-design-system/styles/dist/index.css" />
+  </head>
+  <body>
+  </body>
+</html>
+```
+
+We also provide a default export that points to the created `dist/index.css` file.
+This will be picked up and served when using a bundler (e.g. webpack or vite).
+For this to work, import the module directly in your project like this.
+
+```javascript
+import "@synergy-design-system/styles";
+```
+
+---
+
+### 2.2 Loading single modules
+
+It is also possible to load only parts of the provided functionality.
+This can be archived in the following way:
+
+```html
+<!-- Loading typography only -->
+<!DOCTYPE html>
+  <head>
+    <link rel="stylesheet" href="/node_modules/@synergy-design-system/styles/dist/typography.css" />
+  </head>
+  <body>
+  </body>
+</html>
+```
+
+```javascript
+// Loading typography only
+import "@synergy-design-system/styles/typography.css";
+```
+
+---
+
+## 3. Available modules
+
+<!-- BEGIN INLINE COMMENT -->
+
+### 3.1 - typography
+
+#### Usage
+
+```html
+<link
+  rel="stylesheet"
+  href="/node_modules/@synergy-design-system/styles/dist/typography.css"
+/>
+```
+
+```javascript
+import "@synergy-design-system/styles/typography.css";
+```
+
+#### Submodules
+
+- body
+- heading
+- weight
+
+<!-- END INLINE COMMENT -->
+
+---
+
+## Documentation
```

Comment:
```suggestion
## Development
```

Reply:
As the information detailed here is always needed for users when they want to create new modules, I would prefer to leave it in the main Readme as people tend to read the `README.md` of a project only, if they read it at all.

Reply:
After discussion, we aim for a more holistic approach that will be handled in https://github.com/synergy-design-system/synergy-design-system/issues/509. We leave the documentation as is until we have chosen how to procede for all packages.